### PR TITLE
feat(console): add deletion confirm for in-used passwordless connectors

### PR DESCRIPTION
**Pull request by xiaoyijun**
## Summary

feat(console): add deletion confirm for in-used passwordless connectors

## Testing

image

image

image

## Checklist

- [x] `.changset-staged`
- [ ] unit tests
- [ ] integration tests
- [ ] docs

OR

- [x] This PR is not applicable for the checklist

### CODE REVIEW

File: packages/console/src/pages/ConnectorDetails/components/DeleteConnectorConfirmModal/index.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  onCancel: () => void;
+  onConfirm: () => void;
+};
+
+const DeleteConnectorConfirmModal = ({ data, isOpen, onCancel, onConfirm }: Props) => {
+  const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
+  const isSocial = data.type === ConnectorType.Social;
+
+  return (
+    <ConfirmModal
```
Comment (simeng-li) [inline packages/console/src/pages/ConnectorDetails/components/DeleteConnectorConfirmModal/index.tsx:20]: nit: I have put a global shared ConfirmModal provider.  simply use `useConfirmModal`. No need to build a separate confirm model everywhere.  Not a blocker, please create an issue to optimize later.

File: .changeset-staged/flat-guests-bathe.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+"@logto/console": minor
+"@logto/phrases": minor
+---
```
Comment (gao-sun) [inline .changeset-staged/flat-guests-bathe.md:4]:
```suggestion
"@logto/console": patch
"@logto/phrases": patch
---
```

File: packages/console/src/pages/ConnectorDetails/components/DeleteConnectorConfirmModal/index.tsx [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+      isOpen={isOpen}
+      confirmButtonText="general.delete"
+      onCancel={onCancel}
+      onConfirm={onConfirm}
+    >
+      {isSocial ? (
+        <Trans
+          t={t}
+          i18nKey="connector_details.in_used_social_deletion_description"
```
Comment (gao-sun) [inline packages/console/src/pages/ConnectorDetails/components/DeleteConnectorConfirmModal/index.tsx:29]:
```suggestion
          i18nKey={`connector_details.in_used_${isSocial ? 'social' : 'passwordless'}_deletion_description`}
```

File: packages/phrases/src/locales/de/translation/admin-console/connector-details.ts [hunk 4]
```diff
@@ -23,8 +23,10 @@ const connector_details = {
   type_email: 'E-Mail connector',
   type_sms: 'SMS connector',
   type_social: 'Social connector',
-  in_use_deletion_description:
+  in_used_social_deletion_description:
     'This connector is in use in your sign in experience. By deleting, <name/> sign in experience will be deleted in sign in experience settings.', // UNTRANSLATED
```
Comment (gao-sun) [inline packages/phrases/src/locales/de/translation/admin-console/connector-details.ts:27]:
```suggestion
    'This connector is in-use in your sign in experience. By deleting, <name/> sign in experience will be deleted in sign in experience settings.', // UNTRANSLATED
```

File: packages/phrases/src/locales/en/translation/admin-console/connector-details.ts [hunk 5]
```diff
@@ -23,8 +23,10 @@ const connector_details = {
   type_email: 'Email connector',
   type_sms: 'SMS connector',
   type_social: 'Social connector',
-  in_use_deletion_description:
+  in_used_social_deletion_description:
     'This connector is in use in your sign in experience. By deleting, <name/> sign in experience will be deleted in sign in experience settings.',
+  in_used_passwordless_deletion_description:
+    'This {{name}} is in use in your sign-in experience. By deleting, your sign-in experience won’t go live until resolving the conflict.',
```
Comment (gao-sun) [inline packages/phrases/src/locales/en/translation/admin-console/connector-details.ts:29]: is this "won't go live" or "broken?"

**Review by simeng-li — APPROVED** [review]
overall LGTM

**Review by gao-sun — APPROVED** [review]

File: packages/console/src/pages/ConnectorDetails/components/DeleteConnectorConfirmModal/index.tsx (hunk 1 above)
Comment (xiaoyijun) [reply packages/console/src/pages/ConnectorDetails/components/DeleteConnectorConfirmModal/index.tsx:20]: LOG-5792

File: .changeset-staged/flat-guests-bathe.md (hunk 2 above)
Comment (xiaoyijun) [reply .changeset-staged/flat-guests-bathe.md]: updated.

File: packages/console/src/pages/ConnectorDetails/components/DeleteConnectorConfirmModal/index.tsx (hunk 3 above)
Comment (xiaoyijun) [reply packages/console/src/pages/ConnectorDetails/components/DeleteConnectorConfirmModal/index.tsx:29]: updated.

File: packages/phrases/src/locales/de/translation/admin-console/connector-details.ts (hunk 4 above)
Comment (xiaoyijun) [reply packages/phrases/src/locales/de/translation/admin-console/connector-details.ts]: updated.

File: packages/phrases/src/locales/en/translation/admin-console/connector-details.ts (hunk 5 above)
Comment (xiaoyijun) [reply packages/phrases/src/locales/en/translation/admin-console/connector-details.ts]: updated with new content.

**Review by gao-sun — APPROVED** [review]

**Pull request merged**